tailwind.config.cjs

```diff
@@ -6,6 +6,19 @@ module.exports = {
   ],
   theme: {
     extend: {},
+    screens: {
+			xl: { max: "1279px" },
+			// => @media (max-width: 1279px) { ... }
+
+			lg: { max: "1023px" },
+			// => @media (max-width: 1023px) { ... }
+
+			md: { max: "767px" },
+			// => @media (max-width: 767px) { ... }
+
+			sm: { max: "639px" },
+			// => @media (max-width: 639px) { ... }
```

Comment:
why are redefining the screens properties?

Reply:
I am using tailwind desktop first,